libleidenalgConfig.cmake.in

```diff
@@ -1,21 +1,22 @@
 # - Config file for the libleidenalg package
 # It defines the following variables
-#  LIBLEIDENALG_INCLUDE_DIRS - include directories for libleidenalg
+#  LIBLEIDENALG_INCLUDEDIR - include directories for libleidenalg
 #  LIBLEIDENALG_LIBRARIES    - libraries to link against
 
-set(LIBLEIDENALG_VERSION 0.9.0)
+set(LIBLEIDENALG_VERSION "@PACKAGE_VERSION_BASE@")
+  if (CMAKE_BUILD_TYPE EQUAL "RELEASE")
+      @PACKAGE_INIT@
 
-@PACKAGE_INIT@
+  # Compute paths
+  set_and_check(LIBLEIDENALG_INCLUDEDIR "@PACKAGE_CMAKE_INSTALL_INCLUDEDIR@")
 
-# Compute paths
-set_and_check(LIBLEIDENALG_INCLUDEDIR "@PACKAGE_LIBLEIDENALG_INCLUDE_DIR")
+  # Our library dependencies (contains definitions for IMPORTED targets)
+  if(NOT TARGET libleidenalg AND NOT LIBLEIDENALG_BINARY_DIR)
+    include("${LIBLEIDENALG_CMAKE_DIR}/libleidenalgTargets.cmake")
```

Comment:
Why is this `include("${LIBLEIDENALG_CMAKE_DIR}/libleidenalgTargets.cmake")` necessary?

Reply:
It allows something to pull in the targets we import, which to my understanding is something that's necessary for a config-based package add.

Unless this is something also being generated elsewhere?

Reply:
OK, finally read up on it, and I believe you are right, and the include should be present indeed.